spec/02-integration/09-hybrid_mode/11-status_spec.lua

```diff
@@ -74,9 +74,6 @@ for _, strategy in helpers.each_strategy() do
     end)
 
     describe("dp status ready endpoint for no config", function()
```

Comment:
Similar as `describe_rpc_sync_on`, could we change it to `describe_rpc_sync_off` and remove `it_no_rpc_sync`?